CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 cmake_minimum_required(VERSION 2.6.4)
 project(CFE_SAMPLE_APP C)
 
+include_directories(../../cfe/fsw/cfe-core/src/tbl)
```

Comment:
This should probably be:
    include_directories(${cfe-core_MISSION_DIR}/src/tbl)